app/views/projects/show.html.erb

```diff
@@ -0,0 +1,9 @@
+name: <%= @project.name %>
+matterial: <%= @project.material %>
+Number of contestants: <%= @project.contestants.count %>
+Average Contestant Experience: <%= @project.average_experience.to_f %> years
+<br>
+"Add Contestant To Project"
+<form action="/projects/<%= @project.id %>/contestants" method="post">
```

Comment:
Nice work with this extension! You create a very RESTful route here for this POST